src/PageContent/Settings/Settings.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react';
+import './StudentCoordinatorSettings.css';
+import OrganizationDetails from './OrganizationDetails';
+import AccountManager from './AccountManager';
+import './SCSettings0.css';
+import './SCSettings1.css';
+
+class Settings extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            account: null,
+            org: null,
+            coordinator: null,
+            secondaryContact: null,
+            isEditingOrg: false,
+            isEditingAcc: false,
```

Comment:
the `isEditing` flags don't need to be in this level? They could just be inside the sub-content components themselves?

Reply:
Fixed.